### PR TITLE
chore: linear cut conf and store dependency

### PR DESCRIPTION
**Pull request by Yisaer**

### CODE REVIEW

File: internal/conf/logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	filename "github.com/keepeye/logrus-filename"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	Log     *logrus.Logger
+	LogFile *os.File
+)
+
+func init() {
```
Comment (ngjaying) [inline internal/conf/logger/logger.go:30]: Why adding init as InitLogger already be invoked explicitly.

File: internal/conf/time.go [hunk 2]
```diff
@@ -18,11 +18,19 @@ import (
...
 
 	"github.com/benbjohnson/clock"
+
+	"github.com/lf-edge/ekuiper/internal/conf/logger"
 )
 
 var Clock clock.Clock
 
 func InitClock() {
+	if logger.Log == nil {
```
Comment (ngjaying) [inline internal/conf/time.go:28]: Can we say conf init() already guarantee logger exists? So this is unnecessary.

**Review by ngjaying — CHANGES_REQUESTED** [review]

File: internal/conf/logger/logger.go (hunk 1 above)
Comment (Yisaer) [reply internal/conf/logger/logger.go:30]: In some unit test, `InitLogger` won't be called by `conf.init` as expected.

Comment (ngjaying) [reply internal/conf/logger/logger.go:30]: It looks like current implementation will init logger twice. Can we just remove InitLogger in conf?

Comment (Yisaer) [reply internal/conf/logger/logger.go:32]: updated.

**Review by ngjaying — APPROVED** [review]

**Pull request merged**